### ISSUE TITLE
riak (2.0.5)/riak cs(2.0.1) interaction - riak config effective crashes on <<"0b:">> [JIRA: RCS-302]

### ISSUE THREAD

**Issue by binarytemple** [basho/riak_cs#1254]
[/basho/riak/dev/dev1%]git status
HEAD detached at riak-2.0.5

[/basho/riak/dev/dev1%]./bin/riak config effective
escript: exception error: bad argument
  in function  atom_to_list/1
     called as atom_to_list(<<"0b:">>)
  in call from cuttlefish_effective:'-proplist_to_kvcpaths/2-fun-0-'/4 (src/cuttlefish_effective.erl, line 124)
  in call from lists:foldl/3 (lists.erl, line 1248)
  in call from cuttlefish_effective:'-proplist_to_kvcpaths/2-fun-0-'/4 (src/cuttlefish_effective.erl, line 125)
  in call from lists:foldl/3 (lists.erl, line 1248)
  in call from cuttlefish_effective:'-proplist_to_kvcpaths/2-fun-0-'/4 (src/cuttlefish_effective.erl, line 125)
  in call from lists:foldl/3 (lists.erl, line 1248)
  in call from cuttlefish_effective:process_advanced/2 (src/cuttlefish_effective.erl, line 97)

**Comment by binarytemple** [comment]
In this example, can be worked around by editing the advanced.config to remove the multi-backend section, e.g. 

```
[{riak_kv, [
%%    {add_paths, ["/basho/riak_cs/dev/dev1/lib/riak_cs-2.0.1/ebin"]},
%%    {storage_backend, riak_cs_kv_multi_backend},
%%    {multi_backend_prefix_list, [{<<"0b:">>, be_blocks}]},
%%    {multi_backend_default, be_default},
%%    {multi_backend, [
%%        {be_default, riak_kv_eleveldb_backend, [
%%            {total_leveldb_mem_percent, 30},
%%            {data_root, "/basho/riak/dev/dev1/data/leveldb"}
%%        ]},
%%        {be_blocks, riak_kv_bitcask_backend, [
%%            {data_root, "/basho/riak/dev/dev1/data/bitcask"}
%%        ]}
%%    ]}
]}].
```

**Comment by binarytemple** [comment]
Referenced this issue in the [cuttlefish issue tracker](https://github.com/basho/cuttlefish/issues/203) as well.

**Comment by shino** [comment]
This is a bug of cuttlefish. Close.

**Comment by binarytemple** [comment]
Yes, probably better off being fixed there, I wasn't sure who took responsibility for maintaining the cuttlefish mapping.

**Comment by shino** [comment]
Understood. It's (almost) riak_cs specific and this issue is nice place to track or not to forget.
